onnx_tf/backend.py

```diff
@@ -390,7 +382,7 @@ def onnx_graph_to_tensorflow_net(cls, graph_def):
     # defined tensors so we can have access to the placeholders
     # to feed in input tensors when we run the graph.
     original_input_dict = dict(input_dict_items)
-    output_dict = dict()
+    output_dict = input_dict
```

Comment:
Is this necessary?
Does not look very reassuring...

Reply:
@tjingrant 
For `test_operator_chunk`, no node in graph and input is just output. Just for that case.